### PR TITLE
Fix Duration Selector Default

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

Sets the value via Value Changed event on first updated. This make sure that HA form is aware without changing any lower element functionality 

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-selector/ha-selector-duration.ts [hunk 1]
```diff
@@ -18,6 +19,19 @@ export class HaTimeDuration extends LitElement {
 
   @property({ type: Boolean }) public required = true;
 
+  protected firstUpdated(changedProps: PropertyValues): void {
```
Comment (zsarnett) [inline src/components/ha-selector/ha-selector-duration.ts:22]: I am thinking now. Should this be in will update?

Comment (balloob) [reply src/components/ha-selector/ha-selector-duration.ts:22]: `ha-form` applies the default value if nothing is set. It's weird that we can get here and then set the duration.

I think instead in the rendering we should just do `const value = this.value || "0:00:00"`

Comment (zsarnett) [reply src/components/ha-selector/ha-selector-duration.ts:22]: We need to send the value to HA form

Comment (zsarnett) [reply src/components/ha-selector/ha-selector-duration.ts:22]: Ha form doesnt default the value.

Not for selectors

Comment (balloob) [reply src/components/ha-selector/ha-selector-duration.ts:22]: Fix it here https://github.com/home-assistant/frontend/blob/dev/src/components/ha-form/compute-initial-ha-form-data.ts#L3

**Review by balloob — APPROVED** [review]

**Pull request merged**